relay-general/src/store/normalize/contexts.rs

```diff
@@ -215,6 +226,21 @@ mod tests {
     use super::*;
     use crate::protocol::LenientString;
 
+    #[test]
+    fn test_get_product_name() {
```

Comment:
this test feels a bit silly now since it's only checking that the values in the hashmap exist and nothing more

Reply:
Well, now it's also checking the headers are passed correctly 😛.